integration/Dockerfile

```diff
@@ -1,11 +1,11 @@
-
 ARG CONTAINERD_VER=1.6.15
 ARG CONTAINERD_PROJECT=/containerd
 ARG RUNC_VERSION=1.1.4
 ARG NYDUS_SNAPSHOTTER_PROJECT=/nydus-snapshotter
 ARG DOWNLOADS_MIRROR="https://github.com"
 ARG NYDUS_VER=2.2.4
-ARG NERDCTL_VER=1.0.0
+ARG NERDCTL_VER=1.7.6
+ARG DELVE_VER=1.23.0
```

Comment:
Pinning delve will avoid one of the recent breakage of the CI, where the latest delve started depending on go features only available in a more recent version of go that what we used here.